examples/aws_efa/nccl_efa.yaml

```diff
@@ -50,6 +66,8 @@ config:
         containers:
         - resources:
             limits:
-              vpc.amazonaws.com/efa: 2
+              vpc.amazonaws.com/efa: 4
```

Comment:
Can we update the README for how to setup this number for different number of GPUs and GPU type?